### PR TITLE
Update gcm docs with latest timelines

### PR DESCRIPTION
**Pull request by Bucimis**

### CODE REVIEW

File: _docs/_developer_guide/platform_integration_guides/android/push_notifications/integration.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ For devices without Google services installed, Braze offers the option to send p
 Use [Firebase Cloud Messaging](https://firebase.google.com/docs/cloud-messaging/) (FCM) to register for push.
 
 {% alert update %}
-Though you could previously use GCM, Google has announced that they will remove support for GCM as soon as April 11, 2019 and automatic GCM registration is unavailable through the Braze SDK. If your app is currently supporting GCM, we advise that you speak to your development teams about transitioning to [Firebase from GCM](https://developers.google.com/cloud-messaging/android/android-migrate-fcm) as soon as possible.
+Though you could previously use GCM, Google removed support for GCM on May 29, 2019 and automatic GCM registration is unavailable through the Braze SDK. If your app is currently supporting GCM, we advise that you speak to your development teams about transitioning to [Firebase from GCM](https://developers.google.com/cloud-messaging/android/android-migrate-fcm) as soon as possible.
```
Comment (radixdev) [inline _docs/_developer_guide/platform_integration_guides/android/push_notifications/integration.md]: I'd reword the first sentence (run-on & passive voice). Maybe something like:
`Automatic GCM registration is unavailable through the Braze SDK as a result of Google's removal of support for GCM on May 29, 2019`

Comment (Bucimis) [reply _docs/_developer_guide/platform_integration_guides/android/push_notifications/integration.md]: fixed 👍

**Comment by Bucimis** [comment]
@EmilyNecciai please merge
